docs/learn/tour/tables.md

```diff
@@ -1,209 +1,307 @@
 ---
-title: Tables in kdb+ | A whirlwind tour of kdb+ and q | Documentation for kdb+ and the q programming language
-author: Stephen Taylor
-date: February 2020
-description: How to construct, manipulate and save tables in kdb+
+title: Tables in kdb+ 
+author: Conor McCarthy
+date: December 2024
+description: A brief tour of tables in kdb+/q
 ---
-# Tables
+# Tables in kdb+
 
+This page introduces you to the types of tables which exist in q, how these tables are created and what makes tables in q so powerful.
 
+Unlike in many other programming languages where the tables you encounter are added as a second-class extension to the language, in q they are first-class objects. 
 
```

Comment:
@cmccarthy1 i would add here the "What is a table?" part (lines 27-31, until "interactions with time-series data.)